custom/icds_reports/migrations/sql_templates/create_functions.sql

```diff
@@ -1166,6 +1190,9 @@ BEGIN
 	all_text = 'All';
 	null_value = NULL;
 
+    -- should there be indexes created on this table?
+    -- it doesn't appear to be filtered in this file. maybe tableau touches it?
+
```

Comment:
This table is used by Tableau (there is a join between this table and any of the monthly tables for Tableau queries).